### ISSUE TITLE
"gradle drivers" failed

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/inspectus4katalon-sample-project#47]
```
~/katalon-workspace/MyVisualInspectionProject $ gradle drivers
> Task :drivers FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':drivers'.
> Could not resolve all files for configuration ':Inspectus'.
   > Could not find com.kazurayam:ExecutionProfilesLoader:2.1.0.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/kazurayam/ExecutionProfilesLoader/2.1.0/ExecutionProfilesLoader-2.1.0.pom
       - file:/Users/kazuakiurayama/.m2/repository/com/kazurayam/ExecutionProfilesLoader/2.1.0/ExecutionProfilesLoader-2.1.0.pom
     Required by:
         project :

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 7s
```

**Comment by kazurayam** [comment]
The ExecutionProfilesLoader v2.1.0 has not been published to the Maven Central repository.


<img width="1019" alt="Screenshot 2023-10-27 at 6 34 37" src="https://github.com/kazurayam/inspectus4katalon-sample-project/assets/4467417/2822f184-1076-4d88-96e9-43815601c5f5">

**Comment by kazurayam** [comment]
I have published the v2.1.0 of ExecutionProfilesLoader

https://mvnrepository.com/artifact/com.kazurayam/ExecutionProfilesLoader/2.1.0

**Comment by kazurayam** [comment]
In the inpectus4katalon-sample project,
```
$ gradle drivers
```
succeeded.
